Project/Src/Gui/TextView.cs

```diff
@@ -1062,17 +1062,43 @@ private void DrawTabMarker(Graphics g, Color color, int x, int y)
             DrawString(g, "\u00BB", tabMarkerColor.GetFont(TextEditorProperties.FontContainer), color, x, y);
         }
 
-        private int DrawEOLMarker(Graphics g, Color color, Brush backBrush, int x, int y)
+        private int DrawEOLMarker(EndOfLineType eolType, Graphics g, Color color, Brush backBrush, int x, int y)
         {
             var eolMarkerColor = textArea.Document.HighlightingStrategy.GetColorFor("EOLMarkers");
 
-            var width = GetWidth(ch: '\u00B6', eolMarkerColor.GetFont(TextEditorProperties.FontContainer));
+            int eolMarkerWidth = 0;
+            string representation = "";
+
+            int backslashWidth = GetWidth(ch: '\\', eolMarkerColor.GetFont(TextEditorProperties.FontContainer));
+            int nWidth = GetWidth(ch: 'n', eolMarkerColor.GetFont(TextEditorProperties.FontContainer));
+            int rWidth = GetWidth(ch: 'r', eolMarkerColor.GetFont(TextEditorProperties.FontContainer));
+            switch (eolType)
+            {
+                case EndOfLineType.CarriageReturn:
+                    eolMarkerWidth = backslashWidth + rWidth;
+                    representation = "\\r";
+                    break;
+                case EndOfLineType.CarriageReturnLineFeed:
+                    eolMarkerWidth = backslashWidth + rWidth + backslashWidth + nWidth;
+                    representation = "\\r\\n";
+                    break;
+                case EndOfLineType.LineFeed:
+                    eolMarkerWidth = backslashWidth + nWidth;
+                    representation = "\\n";
```

Comment:
Just asking for opinions, @gitextensions/git-extensions-team:
My favorite diff tool uses the - more or less arbitrary but perhaps common - symbols
![Cr](https://user-images.githubusercontent.com/36601201/136460983-5c77d431-af44-430d-8bf6-c38a42bbeb43.png)
![CrLf](https://user-images.githubusercontent.com/36601201/136461048-33793f20-bc91-458d-8ef4-5ce8920a3035.png)
![Lf](https://user-images.githubusercontent.com/36601201/136461184-a3cf6522-6d01-4aef-8637-c07021afcc5b.png)
Though "\r\n" is more clear for C* programmers like us - but takes more space and might raise questions from other users.

Reply:
As an alternative, if some people don't like `\r` and `\n`, maybe render `CR`, `LF` and `CRLF` instead?

Reply:
@RussKie Notepad++ uses CR and LF glyphs, I do not think they are part of any official font but most likely a blitted bitmap, is my guess.

![image](https://user-images.githubusercontent.com/14078392/136662061-c5829f01-a244-4709-a44c-f54e654c55a3.png)

I would rather keep it simple, if we were to choose to render CR, LF, and CRLF, I would just go for basic text rendering (which might look a little ugly compared to np++).  Personally, I would go with `\r`, `\n`, `\r\n` as it is the most clear (and like you said, programmers will usually know what these mean, and Git was designed for programmers 😄).

If you truly want, we can expose a user option so that they can choose between different styles of EOL representation (non-discriminating pilcrow [the current glyph] vs \r and \n vs @mstv's suggested glyphs or CR, LF)